pkgs/top-level/all-packages.nix

```diff
@@ -16457,6 +16457,11 @@ with pkgs;
 
   pythonmagick = callPackage ../applications/graphics/PythonMagick { };
 
+  python-language-server = callPackage ../servers/python-language-server {
```

Comment:
You can use `python3Packages.callPackage`, which will make the python attribute set available automatically.

Reply:
thanks for the tip, it has bothered me for a while to list packages manually.